main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/Real-Dev-Squad/wisee-backend/routes"
+)
+
+func main() {
+	port := flag.String("port", ":8080", "server address to listen on")
+	flag.Parse()
```

Comment:
how about keeping the flag as -p and the default value can be just the number rather than the string `:8080` (ignoring the colon (:))

Reply:
- I was not sure if `-p` is a flag already conflicting with the go cli commands, so to be explicit it's kept `port`. 
- I can do the `:` change but again it would require us to concatenate the port number with `:` which I am not sure is that required, what do you think?